ReadMe.md

```diff
@@ -87,6 +87,7 @@ To copy and paste | コピペ（する） | |
 English | Japanese | Kana | Notes
 --------|----------|------|------
 Device, terminal | 端末 | たんまつ | Smartphones, tablets, e-readers, command-line interfaces, etc
+Cell phone | ケータイ / 携帯電話　| けいたいでんわ | standard (non-smartphone) cell phone　
```

Comment:
Is katakana really used as a common writing style?
Also, are you sure smartphones are *not* referred to as keitai?